pyoos/collectors/collector.py

```diff
@@ -48,7 +51,7 @@ def get_bbox(self):
 
     def set_bbox(self, bbox):
         if bbox is not None:
-            if isinstance(bbox, list) or isinstance(bbox, tuple):
+            if isinstance(bbox, (list, tuple, np.array)) and len(bbox) == 4:
```

Comment:
Allow for numpy arrays and check for the right length. We can probably do better by just trying to make it a `tuple` and checking for `minx, miny, maxx, maxy`. Maybe I  should try that...